x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -208,12 +235,17 @@ func createCloudWatchEvents(getMetricDataOutput *cloudwatch.GetMetricDataOutput,
 		mapOfMetricSetFieldResults[metricKey[0]] = fmt.Sprint(output.Values[0])
 	}
 
+	if len(mapOfMetricSetFieldResults) <= 3 {
+		info = "Missing Cloudwatch data for instance " + instanceID + ". It is expected if this is a new instance. " +
+			"If not, please recheck the period setting in config."
```

Comment:
```suggestion
			"If this shows up multiple times, please recheck the period setting in config."
```
What I miss here, is that we tell the user how to fix it by setting it to 300 or 60s but not sure how to have it in a short message.